otherlibs/Makefile.otherlibs.common

```diff
@@ -117,12 +117,12 @@ endif
 	  "$(INSTALL_LIBDIR_LIBNAME)/"
 ifeq "$(INSTALL_SOURCE_ARTIFACTS)" "true"
 	$(INSTALL_DATA) \
-	  $(CMIFILES:.cmi=.mli) \
+	  $(addprefix $(VPATH)/,$(CMIFILES:.cmi=.mli)) \
```

Comment:
How do the `(addprefix $(VPATH)/,...)` calls work when `$(VPATH)` contains several directories? (Or can this never happen?)

Reply:
Yes, it could happen, even though it complicates things, so it is recommended not to. This particular instance was a left-over from an earlier iteration. Fixed, thanks.